src/app/(admin)/admin/_utils/common.js

```diff
@@ -12,6 +12,19 @@ export function toConnectList(list, cb) {
   return list?.map(id => ({ id: cb?.(id) ?? id })) ?? [];
 }
 
+export function transformTherapyPrices(therapies, therapyPrices) {
+  const result = [];
```

Comment:
🟢 that looks like a great use case for `filter` + `map` instead of foreact

Reply:
Done